tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_invalid_zip_codes_test.py

```diff
@@ -0,0 +1,159 @@
+                observation_date, observation_type_concept_id, observation_source_concept_id, value_as_number, 
+                value_as_string, value_source_concept_id, value_source_value)
+            VALUES
+                -- valid zip codes, will not be affected --
+                (1, 1, 0, date('2019-03-03'), 0, 1585250, null, '12345', null, null), 
+                (2, 2, 0, date('2019-03-03'), 0, 1585250, null, '23456-1234', null, null),
+                
+                -- invalid zip codes, will be sandboxed and updated in observation table --
+                (3, 3, 0, date('2019-03-03'), 0, 1585250, null, '1234', null, null), 
+                (4, 4, 0, date('2019-03-03'), 0, 1585250, null, '123', null, null),
+                (5, 5, 0, date('2019-03-03'), 0, 1585250, null, '12', null, null),
+                (6, 6, 0, date('2019-03-03'), 0, 1585250, null, '1', null, null),
+                
+                -- not zip codes (not zip code concept id), will not be affected --
+                (7, 7, 0, date('2019-03-03'), 0, 1111111, null, '111', null, null),
+                (8, 8, 0, date('2019-03-03'), 0, 2222222, null, '222', null, null)
+            """).render(fq_dataset_name=self.fq_dataset_name)
+
+        self.load_test_data([zip3_lookup_tmpl, observation_tmpl])
+
+        tables_and_counts = [{
+            'fq_table_name':
+                '.'.join([self.fq_dataset_name, 'observation']),
+            'fq_sandbox_table_name':
+                self.fq_sandbox_table_names[0],
+            'fields': [
+                'observation_id', 'person_id', 'observation_concept_id',
+                'observation_date', 'observation_type_concept_id',
+                'observation_source_concept_id', 'value_as_number',
+                'value_as_string', 'value_source_concept_id',
+                'value_source_value'
+            ],
+            'loaded_ids': [1, 2, 3, 4, 5, 6, 7, 8],
+            'sandboxed_ids': [3, 4, 5, 6],
+            'cleaned_values': [
+                (1, 1, 0, self.date, 0, 1585250, None, '12345', None, None),
+                (2, 2, 0, self.date, 0, 1585250, None, '23456-1234', None,
+                 None),
+                (3, 3, 0, self.date, 0, 1585250, 0.0, self.invalid_zip_response,
+                 self.invalid_zip_concept, self.invalid_zip_response),
+                (4, 4, 0, self.date, 0, 1585250, 0.0, self.invalid_zip_response,
+                 self.invalid_zip_concept, self.invalid_zip_response),
+                (5, 5, 0, self.date, 0, 1585250, 0.0, self.invalid_zip_response,
+                 self.invalid_zip_concept, self.invalid_zip_response),
+                (6, 6, 0, self.date, 0, 1585250, 0.0, self.invalid_zip_response,
+                 self.invalid_zip_concept, self.invalid_zip_response),
+                (7, 7, 0, self.date, 0, 1111111, None, '111', None, None),
+                (8, 8, 0, self.date, 0, 2222222, None, '222', None, None)
```

Comment:
I added these extra test scenarios for the cases mentioned in the cleaning rule feedback. [No need to commit unless you want to]
```suggestion
                (8, 8, 0, self.date, 0, 2222222, None, '222', None, None),
                (9, 9, 0, self.date, 0, 1585250, 0.0, self.invalid_zip_response,
                 self.invalid_zip_concept, self.invalid_zip_response),
                (10, 10, 0, self.date, 0, 1585250, 0.0,
                 self.invalid_zip_response, self.invalid_zip_concept,
                 self.invalid_zip_response),
                (11, 11, 0, self.date, 0, 1585250, None, '    12345 ', None,
```